core/src/test/java/be/orbinson/aem/dictionarytranslator/servlets/action/ExportDictionaryServletTest.java

```diff
@@ -71,7 +71,9 @@ void doPostWithDifferentSeparators(String delimiter) throws Exception {
         String expectedContent = "KEY" + delimiter + "en" + delimiter + "nl-BE\n" +
                 "apple" + delimiter + "Apple" + delimiter + "Appel\n" +
                 "banana" + delimiter + "Banana" + delimiter + "Banaan\n" +
-                "cherry" + delimiter + "Cherry" + delimiter + "Kers\n";
+                "cherry" + delimiter + "Cherry" + delimiter + "Kers\n" +
+                "mango" + delimiter + "Mango" + delimiter + "\n" +
+                "papaya" + delimiter + "Papaya" + delimiter + "\n";
```

Comment:
test case for <empty> ?

Reply:
Added with https://github.com/orbinson/aem-dictionary-translator/commit/9dfc27bdcbb54602aa25648730e61591d8c39aa7.